packages/paste-core/components/combobox/stories/index.stories.tsx

```diff
@@ -523,7 +523,8 @@ export const ComboboxControlled = (): React.ReactNode => {
             {item.code} | {item.label} | {item.phone}
           </div>
         )}
-        onInputValueChange={({inputValue}) => {
+        __onControlledAutoCompleteInputValueChangeWorkaround={(e) => {
```

Comment:
question: what if instead of this new prop / API, we remapped `onInputValueChange` to `onChange` always?

Reply:
`onInputValueChange` is the Downshift approved, you should do it this way, API.

It's only because there's a "bug" in Downshift that when it's controlled, you need to circumvent it to stop it taking over the input value.

So we _could_ have it called `onChange` and do a bunch of stuff underneath to make sure it's only `onChange` when it's autocomplete and controlled, otherwise use `onInputValueChange` under the hood.

Reply:
OK thrown alll this away, better approach instead